pkg/sources/github/github.go

```diff
@@ -785,6 +812,12 @@ func (s *Source) handleRateLimit(ctx context.Context, errIn error) bool {
 			retryAfter = retryAfter + jitter
 			rateLimitResumeTime = now.Add(retryAfter)
 			ctx.Logger().Info(fmt.Sprintf("exceeded %s rate limit", limitType), "retry_after", retryAfter.String(), "resume_time", rateLimitResumeTime.Format(time.RFC3339))
+			// Only report the error if a reporter was provided
+			for _, reporter := range reporters {
+				if err := reporter.Err(ctx, fmt.Errorf("exceeded %s rate limit", limitType)); err != nil {
+					ctx.Logger().Error(err, "failed to report rate limit error")
+				}
```

Comment:
Like I mentioned last week, an error returned here doesn't mean it failed to add the error to the report.